constellation_orderboard/migrations/0002_auto_20170216_0902.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
```

Comment:
This file should be renamed to something that isn't the _auto_ name.  It should be safe to rename with no side effects.